src/shadowbox/server/api.yml

```diff
@@ -127,7 +127,60 @@ paths:
       responses:
         '204':
           description: Access key limit deleted successfully.
-
+  /experimental/server/metrics:
+    get:
+      tags: Server
```

Comment:
Tags is a list, so I think this should have been

```yaml
tags:
  - Server
```

See the issue at https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/Jigsaw-Code/outline-server/master/src/shadowbox/server/api.yml